src/test/java/uk/gov/justice/digital/test/SparkTestHelpers.java

```diff
@@ -1,105 +1,66 @@
 package uk.gov.justice.digital.test;
 
 import lombok.val;
-import org.apache.commons.io.FileUtils;
 import org.apache.spark.sql.Dataset;
-import org.apache.spark.sql.Encoders;
 import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SaveMode;
 import org.apache.spark.sql.SparkSession;
-import org.apache.spark.sql.types.DataTypes;
-import org.apache.spark.sql.types.StructField;
-import org.apache.spark.sql.types.StructType;
 import org.hamcrest.Matcher;
 import org.hamcrest.Matchers;
 import org.jetbrains.annotations.NotNull;
 import scala.collection.JavaConverters;
 import scala.collection.Seq;
-import uk.gov.justice.digital.domain.model.TableIdentifier;
 
+import java.io.File;
 import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import static org.hamcrest.Matchers.contains;
 
 public class SparkTestHelpers {
```

Comment:
This is another duplicated class (with some slight differences) where I've moved stuff from the duplicate in to here and deleted the duplicate.